deployment/instascale-clusterrole.yaml

```diff
@@ -50,3 +50,10 @@ rules:
   - update 
   - delete
   - patch
+
+- apiGroups:
+  - ""
+  resources:
+    - secrets
```

Comment:
Didn't we also add a configs resource?

Reply:
No but I can add if it is required, please let me know.

Reply:
@anishasthana could you please confirm if this needs to be updated whenever you have time. Thank you